src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -1,28 +1,67 @@
 package nextstep.subway.line.ui;
 
-import nextstep.subway.line.application.LineService;
-import nextstep.subway.line.dto.LineRequest;
-import nextstep.subway.line.dto.LineResponse;
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.net.URI;
+import nextstep.subway.line.application.LineService;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
 
 @RestController
-@RequestMapping("/lines")
 public class LineController {
-    private final LineService lineService;
+	private final LineService lineService;
+
+	public LineController(final LineService lineService) {
+		this.lineService = lineService;
+	}
+
+	@PostMapping("/lines")
+	public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+		LineResponse line = lineService.saveLine(lineRequest);
+		return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
+	}
+
+	@GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<List<LineResponse>> showLines() {
+		return ResponseEntity.ok(lineService.findAllLines());
+	}
+
+	@GetMapping(value = "/line/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+		return ResponseEntity.ok(lineService.findLineById(id));
+	}
+
+	@PutMapping("/line/{id}")
```

Comment:
```suggestion
	@PutMapping("/lines/{id}")
```

저는 URL 은 복수명사만 사용해야 한다고 생각하는데정훈님은 어떻게 생각하시나요 ? 🤔

Reply:
restful api에서 복수로 표현하는데 실수 했네요.. 감사합니다!